protos/telemetry/telemetry.proto

```diff
@@ -564,13 +564,38 @@ enum FixType {
     FIX_TYPE_RTK_FIXED = 6; // RTK Fixed, 3D position
 }
 
+
+
+
 // Battery type.
 message Battery {
     uint32 id = 3 [(mavsdk.options.default_value)="0"]; // Battery ID, for systems with multiple batteries
     float voltage_v = 1 [(mavsdk.options.default_value)="NaN"]; // Voltage in volts
-    float remaining_percent = 2 [(mavsdk.options.default_value)="NaN"]; // Estimated battery remaining (range: 0.0 to 1.0)
+    float remaining_percent = 2 [(mavsdk.options.default_value)="NaN"]; // Estimated battery remaining (range: 0 to 100)
+    float current_ma = 4 [(mavsdk.options.default_value)="NaN"]; // Current in mA
+    bool status_not_ready_to_use = 5  [(mavsdk.options.default_value)="0"]; // Battery not ready to use
+    bool status_charging = 6  [(mavsdk.options.default_value)="0"]; // Battery charging
+    
+    bool status_cell_balancing = 7  [(mavsdk.options.default_value)="0"]; // Battery is cell balancing (during charging)
```

Comment:
actually, here they all are, so you can tell me if I did it right :-)
These are still all in discussion